### PR TITLE
FE-308: Support for graph editing on web UI

### PR DESCRIPTION
**Pull request by palashkaria**
### Scope & Purpose

This adds back support for editing a graph. 
1. Edge definitions
2. Orphan collections

- [x] :pizza: New feature

### Checklist

- [x] Tests
  - [x] Manually tested
- [x] :book: CHANGELOG entry made

#### Related Information

- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/FE-308

### CODE REVIEW

File: js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/addGraph/ExampleGraphForm.tsx [hunk 1]
```diff
@@ -50,26 +51,39 @@ const exampleGraphsMap = [
...
+    setLoadingState(prev => {
+      return {
+        ...prev,
+        [graphName]: false
+      };
+    });
   };
-  const [isLoading, setIsLoading] = useState(false);
-
+  console.log({ isAnyLoading });
```
Comment (shd8) [inline js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/addGraph/ExampleGraphForm.tsx:86]: Is this `console.log` supposed to remain?

File: js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/listGraphs/EditGraphButtons.tsx [hunk 2]
```diff
@@ -25,23 +30,26 @@ export const EditGraphButtons = ({
...
   };
   return (
     <>
-      <DeleteGraphModal isOpen={isDeleteModalOpen} onClose={onCloseDeleteModal} graph={graph} />
+      <DeleteGraphModal
+        isOpen={isDeleteModalOpen}
+        onClose={onCloseDeleteModal}
+        onSuccess={() => {
+          onClose();
+        }}
```
Comment (shd8) [inline js/apps/system/_admin/aardvark/APP/react/src/views/graphV2/listGraphs/EditGraphButtons.tsx:50]: To maintain consistency with the other `isOpen`, `onClose` ... callbacks.

```suggestion
        onSuccess={onClose}
```

**Review by shd8 — APPROVED** [review]
LGTM, just address a couple optional small comments.

**Comment by palashkaria** [comment]
- [ ] https://jenkins01.arangodb.biz/job/arangodb-ANY-jslint/30656/

**Pull request merged**